src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -342,7 +341,7 @@ instruct vadd_masked(vReg dst_src1, vReg src2, vRegMask_V0 v0) %{
   match(Set dst_src1 (AddVI (Binary dst_src1 src2) v0));
   match(Set dst_src1 (AddVL (Binary dst_src1 src2) v0));
   ins_cost(VEC_COST);
-  format %{ "vadd.vv $dst_src1, $src2, $v0\t#@vadd_masked" %}
+  format %{ "vadd_masked $dst_src1, $src2, $v0" %}
```

Comment:
Suggestion: `format %{ "vadd_masked $dst_src1, $dst_src1, $src2, $v0" %}`

Reply:
Thanks for the review, and has been fixed.